lib/typescript/websocketclient/payload.ts

```diff
@@ -18,6 +18,20 @@ export interface MessageCreatedPayload extends Event {
     };
   };
 }
+export interface MessageUpdatedPayload extends Event {
+  type: 'message.updated';
+  payload: {
+    conversation_id: string;
+    channel_id: string;
+    message: {
+      id: string;
+      content: string;
+      delivery_state: DeliveryState;
+      from_contact: boolean;
+      sent_at: Date;
+    };
+  };
+}
```

Comment:
some types are missing in the payload 🤔: https://airy.co/docs/core/api/websocket#messageupdated